15-random-number-generator/code/README.md

```
@@ -0,0 +1,33 @@
+```
+npm install
+npm run start:watch -- --functionName=function_1 --fixturePath=function_1_event.json
+```
+
+### Adding external dependencies
+You can also add dependencies on external packages in package.json under the "dependencies" key. These dependencies will be made available to your function at runtime and testing.
+
+### Packaging the code
+Once you are done with the testing,
+Run
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)